tempodb/backend/azure/azure_helpers.go

```diff
@@ -32,10 +33,16 @@ func GetContainerURL(ctx context.Context, conf *Config, hedge bool) (blob.Contai
 	}
 
 	var httpSender pipeline.Factory
-	if hedge && conf.HedgeRequestsAt != 0 {
+	if hedge && cfg.HedgeRequestsAt != 0 {
 		httpSender = pipeline.FactoryFunc(func(next pipeline.Policy, po *pipeline.PolicyOptions) pipeline.PolicyFunc {
 			return func(ctx context.Context, request pipeline.Request) (pipeline.Response, error) {
-				client := hedgedhttp.NewClient(conf.HedgeRequestsAt, uptoHedgedRequests, nil)
+
+				customTransport := http.DefaultTransport.(*http.Transport).Clone()
+
+				transport := newInstrumentedTransport(customTransport)
```

Comment:
it appears that users are only going to get metrics if they also hedge requests. it would be preferred to always have metrics regardless of this config option.

Reply:
@joe-elliott Ah yes, I missed that, thanks - updated.